pkgs/by-name/ss/sse2neon/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  stdenv,
```

I've seen a preference for formatting package attributes with a leading comma:

```suggestion
{
, lib
, fetchFromGitHub
, pkg-config
, stdenv
```

---

please see above